docs/signing.rst

```diff
@@ -35,21 +34,21 @@ Signer's perspective (:class:`~nacl.signing.SigningKey`)
     verify_key = signing_key.verify_key
 
     # Serialize the verify key to send it to a third party
-    verify_key_bytes = verify_key.encode()
+    verify_key_hex = nacl.encoding.HexEncoder.encode(bytes(verify_key))
```

Comment:
Seems like we should deprecate `nacl.encoding` entirely? That package only provides interfaces that can be easily replaced by stdlib calls.

Reply:
Yes, but I'd prefer going step by step:
 - signing (almost there?)
 - public
 - secret
 - hash
 - pwhash
 - encoding

Reply:
I think a tracing issue for nacl.encoding deprecation would be useful.

Reply:
Okay, go ahead and file the issue and we can track against that for these items.

Reply:
Opened #532